.circleci/config.yml

```diff
@@ -220,13 +206,23 @@ workflows:
             - wedlocks-tests
             - newsletter-tests
       - build-and-deploy-to-heroku:
+          name: "deploy-locksmith-staging"
           service: locksmith
           app-name: unlock-locksmith-staging
           requires:
             - integration-tests
           filters:
             branches:
-              only: master
+              only: master 
+      - build-and-deploy-to-heroku:
```

Comment:
Do you think we could use the same script and just differentiate between prod and staging using the branches (`production` vs `master`)?

Reply:
you mean move the CI branch logic to the shell script? thats doable